package.json

```diff
@@ -87,7 +89,7 @@
   },
   "devDependencies": {
     "@serverless/eslint-config": "^1.2.0",
-    "@serverless/test": "^2.1.0",
+    "@serverless/test": "^2.3.0",
```

Comment:
Might want to bump this to `2.4.0` to be sure that the newest version is used.

Reply:
It'll be bumped in release PR